src/components/pages/ChatRoom/index.js

```diff
@@ -1,29 +1,21 @@
-import React from 'react';
-import styled from 'styled-components';
+import React , {Component}from 'react';
+import DefaultLayout from '../../molecules/Layout/DefaultLayout';
 
 import ChatNavbar from '../../molecules/ChatNavbar';
 import MessageInput from '../../molecules/MessageInput';
 import MessagesList from '../../organisms/MessagesList';
 
-const Container = styled.div`
-  background: url(/assets/chat-background.jpg);
-  display: flex;
-  flex-flow: column;
-  height: 100vh;
-`;
+class ChatRoom extends Component {
```

Comment:
Is it possible to have functional-based components for testing purposes if we are to add unit tests? Makes it easier to test